decidim-admin/spec/features/participatory_process_admin_manages_participatory_process_admins_spec.rb

```diff
@@ -25,6 +25,7 @@
       switch_to_host(organization.host)
       login_as user, scope: :user
       visit decidim_admin.participatory_process_path(participatory_process)
+      click_link "Process admins"
```

Comment:
Enough for me, but not sure what's your point. What I'm suggesting is just syntax sugar on top of what you put:

```ruby
def within_process_menu do
  within "#tabs-panel" do
    yield
  end
end
```